src/main/java/com/mixpanel/android/viewcrawler/EditState.java

```diff
@@ -159,7 +171,12 @@ public void run() {
             }
             // ELSE View is alive and we are alive
 
-            mEdit.visit(viewRoot);
+            try {
+                mEdit.visit(viewRoot);
+            } catch (ViewVisitor.LayoutUpdateException e) {
+                cleanUp();
+                throw e;
```

Comment:
I'm worried that this rethrow will crash the app. Perhaps LayoutUpdateException shouldn't be a runtime error?